src/include/user/igo_nr.h

```diff
@@ -32,7 +32,7 @@ struct IGO_PARAMS {
 struct sof_igo_nr_config {
 	/* reserved */
 	struct IGO_PARAMS igo_params;
-
+	uint8_t active_channel_idx;
```

Comment:
This is a packed structure so this should be a uint32_t to prevent the remaining data being aligned on a byte.

Reply:
Will fix it. Thanks!